hbase_sql/src/main/java/com/splicemachine/stream/RemoteQueryClientImpl.java

```diff
@@ -122,6 +122,8 @@ public void submit() throws StandardException {
                     streamingBatches, streamingBatchSize, parallelPartitions, shufflePartitionsProperty);
 
             String requestedQueue = (String) activation.getLanguageConnectionContext().getSessionProperties().getProperty(SessionProperties.PROPERTYNAME.OLAPQUEUE);
+            // remove any stale revoked roles (DB-9749)
```

Comment:
We can skip the check of roles if user is database owner, and use the requestedQueue directly.